### ISSUE TITLE
As a developer, I want SequelPro support

### ISSUE THREAD

**Issue by rickmanelius** [drud/ddev#92]
Acceptance Criteria

* Connection details accessible in `ddev describe`
* Documentation on usage.
* Resolution of https://github.com/drud/ddev/issues/81.

**Comment by rickmanelius** [comment]
This is now somewhat of a duplicate of #81 with some additional details regarding where this information should be stored.

**Comment by beeradb** [comment]
I recommend punting this from `v0.4` unless we have other specific services we need to deal with. I With the work we've done in #81 and #194 I feel like we can afford to wait on this for the foreseeable future

**Comment by tannerjfco** [comment]
Is there anything further actionable for this ticket at this point? It seems like it's resolved to me with #81 and #194

**Comment by rickmanelius** [comment]
Agreed. No longer necessary at this time. If we need to re-evaluate, we can re-open or open a new issue.
